loguru.hpp

```diff
@@ -102,6 +102,35 @@ Website: www.ilikebigbits.com
 #include <sal.h>	// Needed for _In_z_ etc annotations
 #endif
 
+/*
+	Will add syslog as a standard sink for log messages
+	Any logging message with a verbosity lower or equal to
+	the given verbosity will be included.
```

Comment:
This is slightly misleading - `LOGURU_SYSLOG` doesn't add a syslog sink, it only enables it.

Maybe this comment was meant for `add_syslog` instead?

Reply:
Moved to loguru.cpp beside the add_syslog() function.